src/ripple/shamap/SHAMapInnerNode.h

```diff
@@ -103,7 +103,7 @@ class SHAMapInnerNode final : public SHAMapTreeNode,
 public:
     explicit SHAMapInnerNode(
         std::uint32_t cowid,
-        std::uint8_t numAllocatedChildren = branchFactor);
+        std::uint8_t numAllocatedChildren = 2);
```

Comment:
What's the motivation for this change? `branchFactor` is 16, but now we are setting this to 2